### PR TITLE
Add ARB

### PR DESCRIPTION
**Pull request by JeremyParish69**
validation successful

### CODE REVIEW

File: packages/web/public/networks/arbitrum.svg
Comment (jonator) [inline packages/web/public/networks/arbitrum.svg:1]: Is there a reason we have 2 of these svg files?

Comment (JeremyParish69) [reply packages/web/public/networks/arbitrum.svg:1]: One for the token in the /tokens/ folder, and one the network in the /networks/ folder. I suppose we might be able to get rid of the /networks/ folder if we just use the corresponding gas token for the network, although I'm not certain whether that may ever need to be different.

Comment (JeremyParish69) [reply packages/web/public/networks/arbitrum.svg:1]: Are we OK to proceed as is? or do you want it removed first?

Comment (jonator) [reply packages/web/public/networks/arbitrum.svg:1]: We can leave it given the difference

**Pull request merged**